draft-ietf-tls-tls13.md

```diff
@@ -1159,13 +1164,17 @@ as with a 1-RTT handshake with PSK resumption.
                                                     + pre_shared_key
                                                         + key_share*
                                                {EncryptedExtensions}
+                                                       + early_data*
                                                           {Finished}
                                  <--------       [Application Data*]
          (EndOfEarlyData)
          {Finished}              -------->
 
          [Application Data]      <------->        [Application Data]
 
+              +  Indicates noteworthy extensions sent in the
+                 previously noted message.
```

Comment:
heh, it was vim yank-and-put, so it is the surrounding text that is inconstent between figures.
But, I will go for local consistency, thanks for pointing it out.